app/assets/stylesheets/recipe_new.scss

```diff
@@ -0,0 +1,101 @@
+@import "colors";
+
+.recipe-name-input-block textarea {
+  font-size: 16pt;
+  font-family: 'Lobster', cursive;
+  background-color: $white;
+  color: $slate;
+  border-color: darken($light-yellow, 5%);
+  border-radius: 5px;
+}
+
+.recipe-attributes-block {
+  padding: 0 5px;
```

Comment:
Properties should be ordered background-color, border-radius, box-shadow, font-size, padding, padding-top